control/controlexecute/result_row.go

```diff
@@ -22,6 +22,21 @@ type ResultRow struct {
 	Control    *modconfig.Control `json:"-" csv:"control_id:UnqualifiedName,control_title:Title,control_description:Description"`
 }
 
+func (r *ResultRow) PassedOrFailed() string {
```

Comment:
what is this for? 
why does it have a lot of hard coded values in

comments and consts please!